sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -128,6 +130,76 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
         ShimLoader.setSparkShimProviderClass(conf.shimsProviderOverride.get)
       }
 
+      try {
```

Comment:
This should be refactored into a private utility method, e.g.: `checkCudfVersion`, so it's easier to see at a high level what `init` needs to do.

Reply:
Thanks @jlowe for your review comments. It looks a lot cleaner now :)